cloudera-on-centos/azuredeploy.json

```diff
@@ -25,24 +25,41 @@
     "cmPassword": {
       "type": "securestring",
       "metadata": {
-        "description": "password for the Cloudera Manager"
+        "description": "password for the Cloudera Manager (A mix of upper and lower-case characters, digits and symbols)"
       }
     },
     "storageAccountPrefix": {
       "type": "string",
       "metadata": {
-        "description": "Unique namespace for the Storage Account where the Virtual Machine's disks will be placed"
+        "description": "Unique namespace for the Storage Account where the VM disks will be placed (alphahumeric characters only)"
       }
     },
     "dnsNamePrefix": {
       "type": "string",
       "metadata": {
-        "description": "Unique public dns name where the Virtual Machines will be exposed"
+        "description": "Unique public DNS name where the VMs will be exposed"
       }
     },
     "region": {
       "type": "string",
-      "defaultValue": "japanwest",
+      "defaultValue": "westeurope",
```

Comment:
Actually, Japan East does have premium storage since August:

https://azure.microsoft.com/en-us/blog/azure-premium-storage-expanding-in-more-regions/

At this point, I believe that we’re on track for the Q4 coverage on that link. I’m deploying on North Europe, which went GA two weeks ago:

https://azure.microsoft.com/en-us/updates/general-availability-azure-premium-storage-in-additional-regions/